v2/internal/runner/runner.go

```diff
@@ -307,6 +311,24 @@ func (r *Runner) RunEnumeration() error {
 		}
 		r.options.Templates = append(r.options.Templates, templatesLoaded...)
 	}
+	if len(r.options.NewTemplatesWithVersion) > 0 {
+		minVersion, _ := semver.Parse("8.8.5")
+		latestVersion, err := semver.Parse(r.templatesConfig.NucleiTemplatesLatestVersion)
+		if err != nil {
+			return errors.Wrap(err, "could not get latest version")
+		}
+		for _, version := range r.options.NewTemplatesWithVersion {
+			current, _ := semver.Parse(strings.Trim(version, "v"))
```

Comment:
`err` should be checked and propagated if not nil

Reply:
Done